tool/gravity/cli/clusterupdate.go

```diff
@@ -96,6 +107,65 @@ func newClusterUpdater(
 	return updater, nil
 }
 
+// checkStatus returns an error if the cluster is degraded.
+// If force is true, warnings will be ignored.
+func checkStatus(ctx context.Context, force bool) error {
+	nodes, err := statusapi.NodeStatuses(ctx)
```

Comment:
If call to `NodeStatuses` returns an error here, upgrade will not be able to continue. Is this okay behavior?